src/webgpu/capability_info.ts

```diff
@@ -77,7 +77,7 @@ export const kErrorScopeFilterInfo: {
 };
 /** List of all GPUErrorFilter values. */
 export const kErrorScopeFilters = keysOf(kErrorScopeFilterInfo);
-export const kGeneratableErrorScopeFilters = kErrorScopeFilters.filter(e => e !== 'internal');
+export const kGeneratableErrorScopeFilters = kErrorScopeFilters.filter(e => e === 'validation');
```

Comment:
This filter trivially results in just one element, so instead I think we should add a column in the info table just above for whether it's generatable. I'll just go ahead and do this.